sdk/src/test/java/io/dapr/client/DaprClientGrpcTest.java

```diff
@@ -2392,6 +2375,7 @@ public void shutdownTest() {
     result.block();
   }
 
+
   private <T> DaprProtos.GetStateResponse buildFutureGetStateEnvelop(T value, String etag) throws IOException {
     return buildGetStateResponse(value, etag);
   }
```

Comment:
Need to bring back the code from previous commit.

Reply:
I confirmed that nothing should change (apart from import organizer) since the new API is in the preview client.